tools/run_net.py

```diff
@@ -31,7 +32,13 @@ def parse_args():
             overwrites the config loaded from file.
         """
     parser = argparse.ArgumentParser(
-        description="Provide SlowFast video training and testing pipeline."
+        description="Provide SlowFast video training, testing, and predicting pipeline."
+    )
+    parser.add_argument(
+        '--predict_source',
+        help='the input source to perform prediction, \
+                can either be a path to a video or index of the camera',
+        default=0,
```

Comment:
Maybe set it to bool rather than int?
`type=bool` with `default=False'

I like the name of data source (or predict_source as you have here), but in order to make it consistent with what we have below (line 103), do you think we could use something like:
```
enable_demo
DEMO.ENABLE
```

Reply:
it's also a good idea if I understand your idea of the `DEMO.ENABLE` correctly, do you mean to have a CfgNode() for it in default.py? if it's so, I'm thinking of having a common Node for demoing prediction, something like this:
```
_C.DEMO = CfgNode()

_C.DEMO.ENABLE = True

_C.DEMO.LABEL_FILE_PATH = ""

_C.DEMO.DATA_SOURCE = 0
```
Instead of:
```
_C.PREDICT = CfgNode()

_C.PREDICT.ENABLE = True

_C.PREDICT.LABEL_FILE_PATH = ""
```
This way I don't have to add predict_source in the parsing arguments in your run_net.py.